[2024-06-28T14:41:44.523+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-28T14:41:44.544+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_transfer_to_gcs.upload_to_gcs manual__2024-06-28T14:36:39.046980+00:00 [queued]>
[2024-06-28T14:41:44.555+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_transfer_to_gcs.upload_to_gcs manual__2024-06-28T14:36:39.046980+00:00 [queued]>
[2024-06-28T14:41:44.555+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-28T14:41:44.565+0000] {taskinstance.py:2330} INFO - Executing <Task(LocalFilesystemToGCSOperator): upload_to_gcs> on 2024-06-28 14:36:39.046980+00:00
[2024-06-28T14:41:44.572+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=651) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-28T14:41:44.573+0000] {standard_task_runner.py:63} INFO - Started process 653 to run task
[2024-06-28T14:41:44.571+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_transfer_to_gcs', 'upload_to_gcs', 'manual__2024-06-28T14:36:39.046980+00:00', '--job-id', '150', '--raw', '--subdir', 'DAGS_FOLDER/to_gcs.py', '--cfg-path', '/tmp/tmpip8d8is7']
[2024-06-28T14:41:44.573+0000] {standard_task_runner.py:91} INFO - Job 150: Subtask upload_to_gcs
[2024-06-28T14:41:44.606+0000] {task_command.py:426} INFO - Running <TaskInstance: data_transfer_to_gcs.upload_to_gcs manual__2024-06-28T14:36:39.046980+00:00 [running]> on host 25c2a2aa63fd
[2024-06-28T14:41:44.688+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_transfer_to_gcs' AIRFLOW_CTX_TASK_ID='upload_to_gcs' AIRFLOW_CTX_EXECUTION_DATE='2024-06-28T14:36:39.046980+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-28T14:36:39.046980+00:00'
[2024-06-28T14:41:44.689+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-28T14:41:44.707+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-06-28T14:41:44.711+0000] {base.py:84} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-06-28T14:41:47.430+0000] {gcs.py:560} INFO - File /opt/***/dags/data/exported_data2.csv uploaded to testing2/data-source.csv in testing-de bucket
[2024-06-28T14:41:47.431+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-28T14:41:47.446+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_transfer_to_gcs, task_id=upload_to_gcs, run_id=manual__2024-06-28T14:36:39.046980+00:00, execution_date=20240628T143639, start_date=20240628T144144, end_date=20240628T144147
[2024-06-28T14:41:47.477+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-28T14:41:47.503+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-28T14:41:47.506+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
